msgraph-mail/php/src/Users/Item/InferenceClassification/Overrides/OverridesRequestBuilder.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Microsoft\Graph\Users\Item\InferenceClassification\Overrides;
+
+use \Exception;
+use Http\Promise\Promise;
+use Http\Promise\RejectedPromise;
+use Microsoft\Graph\Models\Microsoft\Graph\InferenceClassificationOverride;
+use Microsoft\Kiota\Abstractions\HttpMethod;
+use Microsoft\Kiota\Abstractions\MiddlewareOption;
+use Microsoft\Kiota\Abstractions\RequestAdapter;
+use Microsoft\Kiota\Abstractions\RequestInformation;
+use Microsoft\Kiota\Abstractions\ResponseHandler;
+use Microsoft\Kiota\Abstractions\Serialization\Parsable;
+
+class OverridesRequestBuilder 
+{
+    /** @var array|null $pathParameters Path parameters for the request */
+    private array $pathParameters;
+    
+    /** @var RequestAdapter|null $requestAdapter The request adapter to use to execute the requests. */
+    private RequestAdapter $requestAdapter;
+    
+    /** @var string|null $urlTemplate Url template to use to build the URL for the current request builder */
+    private ?string $urlTemplate;
+    
+    /**
+     * Instantiates a new OverridesRequestBuilder and sets the default values.
+     * @param array $pathParameters Path parameters for the request
+     * @param RequestAdapter $requestAdapter The request adapter to use to execute the requests.
+    */
+    public function __construct(array $pathParameters, RequestAdapter $requestAdapter) {
+        $this->urlTemplate = '{+baseurl}/users/{user_id}/inferenceClassification/overrides{?top,skip,search,filter,count,orderby,select,expand}';
+        $this->requestAdapter = $requestAdapter;
+        $this->pathParameters = $pathParameters;
+    }
+
+    /**
+     * A set of overrides for a user to always classify messages from specific senders in certain ways: focused, or other. Read-only. Nullable.
+     * @param GetQueryParameters|null $queryParameters Request query parameters
+     * @param array|null $headers Request headers
+     * @param array|null $options Request options
+     * @return RequestInformation
+    */
+    public function createGetRequestInformation(?GetQueryParameters $queryParameters, ?array $headers, ?array $options): RequestInformation {
```

Comment:
does PHP support optional parameters? in other words, can I call only `createGetRequestInformation()` without getting a compile or runtime error?

Reply:
Yes we can if I set the default value to null.

Reply:
so at the end it'd look something like that?
```suggestion
    public function createGetRequestInformation(?GetQueryParameters $queryParameters = $null, ?array $headers = $null, ?array $options = $null): RequestInformation {
```

Reply:
Yes but the `$null` doesn't have the `$`

Reply:
perfect, let's add that to the generator then

Reply:
Done.